index.html

```diff
@@ -1102,21 +1086,68 @@
           modifiers for particular payment method identifiers. For example, it
           allows you to adjust the total amount based on payment method.
         </dd>
+      </dl>
+    </section>
+    <section data-dfn-for="PaymentDetailsInit" data-link-for=
+    "PaymentDetailsInit">
+      <h2>
+        <dfn>PaymentDetailsInit</dfn> dictionary
+      </h2>
+      <pre class="idl">
+        dictionary PaymentDetailsInit : PaymentDetailsBase {
+          required PaymentItem total;
```

Comment:
Filed https://github.com/heycam/webidl/issues/321. I guess the alternative is to just duplicate the contents into each dictionary instead of using inheritance... sigh.